dbt/include/athena/macros/materializations/models/table/create_table_as.sql

```diff
@@ -52,10 +52,10 @@
     {{ location_property }}='{{ location }}',
   {%- endif %}
   {%- if partitioned_by is not none %}
-    {{ partition_property }}=ARRAY{{ partitioned_by | tojson | replace('\"', '\'') }},
+    {{ partition_property }}=ARRAY{{ partitioned_by | tojson | replace('\"', "'") }},
```

Comment:
could you revert this back please? let's aim to do minimal changes and consider to open a separated pr for that.

Reply:
Sounds good!